src/app/personside/infotabs/meldinger/traadvisning/Enkeltmelding.tsx

```diff
@@ -57,13 +57,16 @@ function journalfortMelding(melding: Melding) {
 function Journalforing({ melding }: { melding: Melding }) {
     const [open, setOpen] = useState(false);
     return melding.journalfortAv && melding.journalfortDato ? (
-        <JournalforingPanel
-            heading={<JournalforingEtikett>Meldingen er journalført</JournalforingEtikett>}
-            apen={open}
-            onClick={() => setOpen(!open)}
-        >
-            {journalfortMelding(melding)}
-        </JournalforingPanel>
+        <div>
+            <hr />
```

Comment:
er ikke helt sikker på hva `hr` betyr rent semantisk, men tror jeg ville stylet dette med css heller enn å legge inn et ekstra element i dom'en..?

Reply:
I html4 så betydde det horizonal-rule, e.g en skillelinje.
Men sjekket akkurat og tydeligvis har definisjonen endret seg i html5. :P 

> Historically, this has been presented as a horizontal rule or line. While it may still be displayed as a horizontal rule in visual browsers, this element is now defined in semantic terms, rather than presentational terms, so if you wish to draw a horizontal line, you should do so using appropriate CSS.